doc/examples/data/plot_3d.py

```diff
@@ -0,0 +1,25 @@
+"""
+==========================================
+datasets with 3 or more spatial dimensions
+==========================================
+
+Most scikit-image functions are compatible with 3-D datasets, ie images with
```

Comment:
```suggestion
Most scikit-image functions are compatible with 3D datasets, i.e., images with
```
(current docs spell "3D" rather than "3-D")